android/src/main/java/com/mybigday/rnmediaplayer/ExternalDisplay.java

```diff
@@ -244,14 +244,20 @@ public void clearScreen() {
   @Override
   public void onHostResume() {
     start();
-    helper.onResume();
+    // Don't know why this would be null at this point
+    if (helper != null) {
+      helper.onResume();
+    }
     showVirtualScreen(true);
   }
 
   @Override
   public void onHostPause() {
     showVirtualScreen(false);
-    helper.onPause();
+    // Don't know why this would be null at this point
+    if (helper != null) {
+      helper.onResume();
```

Comment:
I agree that onPause shouldn't need that verification. I added it just as a precaution. I've never had an NPE pointing to onPause.